packages/core/src/rules/utils/lazy-loading-rule-map.ts

```diff
@@ -0,0 +1,76 @@
+import { Rule } from '../../types/Rule';
```

Comment:
Do this fixes the cyclic dependency that makes all tests trigger when any rule is changed?

Reply:
Oops. I mistakenly added this file! it's an experiment that should be in a different branch, to solve the issue you mentioned :)